lutris/gui/config_dialogs.py

```diff
@@ -452,8 +452,9 @@ def get_config_id(self):
 class EditGameConfigDialog(Dialog, GameDialogCommon):
     """Game config edit dialog."""
     def __init__(self, parent, game, callback):
+        headline_game_config_dialog = _("Configure ") + game.name
         super(EditGameConfigDialog, self).__init__(
-            "Configure %s" % game.name,
+            headline_game_config_dialog,
```

Comment:
So I believe it is best to actually expose `%s` here as it allows RTL languages to be in the correct order. Same for a few others. It does have the downside that bad translations can cause runtime errors but there are tools to validate this.